bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html

```diff
@@ -137,7 +137,7 @@ <h4 class="section-title" id="apps">{{ _('"Apps."') }}</h4>
       <h4 class="section-title" id="services">{{ _('Sharing, Data and Services') }}</h4>
 
       <p>
-        {% trans f1="//f1.mozillamessaging.com/" %}
+        {% trans f1="http://f1.mozillamessaging.com/" %}
```

Comment:
Yes certain URLs just don't exist anymore. I'm not going to try and find fixes for all in this PR, just correct the links that are working but currently pointing to the wrong protocol.